README.md

```diff
@@ -1,37 +1,50 @@
-## TODO
+# Supaapps Auth Guard <!-- omit in toc -->
 
+- [Installation](#installation)
+  - [ENV vars](#env-vars)
+  - [Add new custom guard](#add-new-custom-guard)
+- [Usage example](#usage-example)
+- [Testing](#testing)
 
+## Installation
 
 `composer require supaapps/supaapps-guard`
 
+### ENV vars
 
-env vars:
+add env vars to your `.evn`:
 
-- `SUPAAPPS_GUARD_AUTH_SERVER_URL`
-- `SUPAAPPS_GUARD_AUTH_REALM`
-
-
----
+```env
+`SUPAAPPS_GUARD_AUTH_SERVER_URL=http://example.com`
+`SUPAAPPS_GUARD_AUTH_REALM_NAME=myapp`
+```
 
+### Add new custom guard
 
-### auth.php
+On `config/auth.php` add the new guard
 
 ```php
-
-    'guards' => [
-        'jwt' => [
-            'driver' => 'supaapps-guard',
-            'provider' => 'users',
-        ],
+'guards' => [
+    'jwt' => [
+        'driver' => 'supaapps-guard',
+        'provider' => 'users',
     ],
+],
+```
 
+Also, set the default guard to `jwt`
```

Comment:
+ if not set as default, every-time you use `auth()` remember to pass driver name parameter: `auth('jwt')`